.evergreen/ensure-cmake.sh

```diff
@@ -140,6 +140,9 @@ _ensure_cmake() {
 
     declare arch
     arch="$(uname -m)"
+    if test -f /etc/alpine-release; then
+        arch="$arch-musl"
```

Comment:
Is appending `-musl` for the benefit of logging? It does not appear to impact the cmake or ninja downloads.

Reply:
It causes the `case` branch to fall through to `_build_cmake`/`_build_ninja`, which is required since the prebuilt downloads do not work with muslc.